src/main/java/hu/akarnokd/rxjava/interop/RxJavaInterop.java

```diff
@@ -16,6 +16,9 @@
 
 package hu.akarnokd.rxjava.interop;
 
+import io.reactivex.BackpressureStrategy;
```

Comment:
Please don't use imports in this class so that there is no ambiguity due to naming conflicts.